packages/adapter-static/index.js

```diff
@@ -62,6 +62,7 @@ See https://kit.svelte.dev/docs/page-options#prerender for more details`
 			builder.rimraf(assets);
 			builder.rimraf(pages);
 
+			builder.generateEnvModule();
```

Comment:
If `adapter-static` uses this it needs a major version bump and a SvelteKit 2.x peer dependency bump

Reply:
Oops, good catch. Will fix when I'm at my desk